lib/interface_quda.cpp

```diff
@@ -9178,116 +9103,55 @@ void calcMG_loop_wOneD_TSM_wExact(void **gaugeToPlaquette,
       dirac.prepare(in,out   ,*x   ,*b,param->solution_type); 
```

Comment:
Maybe we should set "out" and "out_LP" to zero, before the "dirac.prepare(....)" to make sure that initial guess is zero for the inversion, i.e.:
zeroCuda(*out);
zeroCuda(*out_LP);